broccoli/packages.js

```diff
@@ -0,0 +1,280 @@
+    annotation: 'ember-glimmer es'
+  });
+}
+
+module.exports.handlebarsES = function _handlebars() {
+  return new Rollup(findLib('handlebars', 'lib'), {
+    rollup: {
+      entry: 'handlebars/compiler/base.js',
+      dest: 'handlebars.js',
+      format: 'es',
+      exports: 'named',
+      plugins: [handlebarsFix()]
+    },
+    annotation: 'handlebars'
+  })
+}
+
+function handlebarsFix() {
+  var HANDLEBARS_PARSER = /\/parser.js$/;
+  return {
+    load: function(id) {
+      if (HANDLEBARS_PARSER.test(id)) {
+        var code = readFileSync(id, 'utf8');
+        return {
+          code: code
+            .replace('exports.__esModule = true;', '')
+            .replace('exports[\'default\'] = handlebars;', 'export default handlebars;'),
+
+          map: { mappings: null }
+        };
+      }
+    }
+  }
+}
+
+module.exports.rsvpES = function _rsvpES() {
+  let lib = path.resolve(path.dirname(require.resolve('rsvp')), '../lib');
+  return new Rollup(lib, {
+    rollup: {
+      entry: 'rsvp.js',
+      dest: 'rsvp.js',
+      format: 'es',
+      exports: 'named'
+    },
+    annotation: 'rsvp.js'
+  });
+}
+
+module.exports.backburnerES = function _backburnerES() {
+  return funnelLib('backburner.js', 'dist/es6', {
+    files: ['backburner.js'],
+    annotation: 'backburner es'
+  });
+}
+
+module.exports.dagES = function _dagES() {
+  return funnelLib('dag-map', {
+    files: ['dag-map.js'],
+    annotation: 'dag-map es'
+  });
+}
+
+module.exports.routeRecognizerES = function _routeRecognizerES() {
+  return funnelLib('route-recognizer', {
+    files: ['route-recognizer.es.js'],
+    getDestinationPath() {
+      return 'route-recognizer.js'
+    },
+    annotation: 'route-recognizer es'
+  });
+}
+
+
+module.exports.simpleHTMLTokenizerES = function _simpleHTMLTokenizerES() {
+  return new Rollup(findLib('simple-html-tokenizer', 'dist/es6'), {
+    rollup: {
+      entry: 'index.js',
+      dest: 'simple-html-tokenizer.js',
+      format: 'es',
+      exports: 'named'
+    },
+    annotation: 'simple-html-tokenizer es'
+  })
+}
+
+module.exports.emberPkgES = function _emberPkgES(name, rollup, externs) {
+  if (rollup) {
+    return new Rollup(`packages/${name}/lib`, {
+      rollup: {
+        entry: 'index.js',
+        dest: `${name}.js`,
+        external: externs,
+        format: 'es',
+        exports: 'named'
+      }
+    });
+  }
+
+  return new Funnel(`packages/${name}/lib`, {
+    exclude: ['.gitkeep'],
+    getDestinationPath(relativePath) {
+      return `${name}/` + relativePath;
+    },
+    annotation: `${name} es`
+  });
+}
+
+module.exports.glimmerPkgES = function _glimmerPkgES(name, externs = []) {
+  return new Rollup(findLib(name, 'dist/modules'), {
+    rollup: {
+      entry: 'index.js',
+      dest: `${name}.js`,
+      external: externs,
+      format: 'es',
+      exports: 'named'
+    },
+    annotation: `${name} es`
+  });
+}
+
+module.exports.emberTestsES = function _emberTestES(name) {
```

Comment:
I'm confused, is this the same as `broccoli/ember-tests-es.js`, do we need both?